src/openrct2/windows/Guest.cpp

```diff
@@ -670,7 +669,7 @@ void window_guest_set_page(rct_window* w, sint32 page){
     w->selected_list_item = -1;
 
     rct_viewport* viewport = w->viewport;
-    w->viewport = 0;
+    w->viewport = nullptr;
```

Comment:
In the `/windows/` folder there are probably a lot of instances of `NULL` that should be replaced with `nullptr` as well

Reply:
I think I got all NULLs from cpp-files now.